beesdoo_stock/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Coop IT Easy SCRLfs
+# Nicolas Jamoulle, <nicolas@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "BEES coop Stock",
+    "version": "12.0.1.0.0",
+    "depends": [
+        'stock',
+    ],
+    "author": "Coop IT Easy SCRLfs",
+    "license": "AGPL-3",
+    "website": "www.coopiteasy.be",
+    "description": """
+        Enable action on multiple products of a stock receipt
+    """,
+    "data": [
+        'views/stock_view.xml',
+    ],
+    'installable': False,
```

Comment:
[Switch the installable key to True](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0#tasks-to-do-in-the-migration)